extra_playbooks/build-rbd-provisioner.yml

```diff
@@ -0,0 +1,55 @@
+---
+
+- hosts: localhost
+  tasks:
+    - name: RBD Provisioner | Install pip packages
+      pip:
+        name: "{{ item.name }}"
+        version: "{{ item.version }}"
+        state: "{{ item.state }}"
+        extra_args: "{{ pip_extra_args | default(omit) }}"
+      with_items:
+        - { state: "present", name: "docker", version: "3.4.1" }
+        - { state: "present", name: "docker-compose", version: "1.21.2" }
+
+    - name: RBD Provisioner | Check Go version
+      shell: |
+        go version
+      ignore_errors: yes
+      register: go_version_result
+
+    - name: RBD Provisioner | Install Go 1.9
```

Comment:
Does not work on container linux

Reply:
Please refer to the above comment (https://github.com/kubernetes-sigs/kubespray/pull/3668#discussion_r237347894).

Reply:
AFAIK this won't work if `localhost` is a non-Ubuntu/Debian machine (t.ex CoreOS/Fedora/Mac).
Either we make that compatible for other platforms or there should be a meaning error message when running on a non supported host, even better if the doc explains it.